causal_testing/testing/causal_test_engine.py

```diff
@@ -162,7 +162,10 @@ def _return_causal_test_results(self, estimator, causal_test_case):
             )
         elif causal_test_case.estimate_type == "risk_ratio":
             logger.debug("calculating risk_ratio")
-            risk_ratio, confidence_intervals = estimator.estimate_risk_ratio()
+            try:
```

Comment:
Is this to catch bad estimate parameters? If so, I'd suggest crashing out with an error rather than falling back on the defaults. If I'd supplied bad arguments, this is the behaviour I'd expect. Otherwise, the user could reasonably expect the returned result to be according to their supplied args when it's actually just from the defaults...

Reply:
This allows for the fact that different estimators implement risk ratios differently and some implementations have no parameters. 

Good point though with it potentially hiding bad parameters and just selecting defaults. I'll look at adding an info logging line if default values are chosen, so there is some visibility?

Reply:
If it's just intended for that, would it be better to `**` the arguments to the estimator constructor? Then if you end up `**`ing nothing, it should still work OK

Reply:
Ahhh that looks a lot cleaner, cheers Micahel! 

I felt that I was missing something..